test/test_GeoMakieExt.jl

```diff
@@ -35,9 +35,39 @@ using OrderedCollections
 
     fig2 = Makie.Figure()
 
-    ClimaAnalysis.Visualize.contour2D_on_globe!(fig2, var2D)
+    ClimaAnalysis.Visualize.contour2D_on_globe!(
+        fig2,
+        var2D,
+        more_kwargs = Dict(
+            :plot => ClimaAnalysis.Utils.kwargs(
+                colormap = Makie.colorschemes[:inferno],
+            ),
+        ),
+    )
 
     output_name = joinpath(tmp_dir, "test_contours2D_globe.png")
     Makie.save(output_name, fig2)
 
+    # Test cmap 
+    test_cmap = ClimaAnalysis.Visualize._constrained_cmap(
+        Makie.colorschemes[:inferno],
```

Comment:
```suggestion
        Makie.colorschemes[:vik],
```

Can you use a divergent colormap so that it is a little easier to see? (same above)